dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/JDBCDriverService.java

```diff
@@ -632,6 +642,12 @@ public Object getDriver(String url) throws SQLException {
                     lock.writeLock().unlock();
                 }
 
+            if(url.toLowerCase().contains("logintimeout") || props.containsKey("loginTimeout")) {
+                SQLNonTransientException failure = connectorSvc.ignoreWarnOrFail(tc, null, SQLNonTransientException.class, "INVALID_LOGINTIMEOUT", dataSourceID);
+                if (failure != null)
+                    throw failure;
+            }
+            
```

Comment:
This code block is copied 3 times, each time right before loadDriver(url, classloader).
Why not move it to the loadDriver method and have it in a single place?

Reply:
Agreed, that would be better.